app/components/account/ConcurrentMerkleTreeCard.tsx

```diff
@@ -0,0 +1,90 @@
+                        <div className="col">
+                            <h3 className="card-header-title">Concurrent Merkle Tree</h3>
+                        </div>
+                    </div>
+                </div>
+
+                <div className="card metadata-json-viewer m-4">
+                    <TableCardBody>
+                        <tr>
+                            <td>Authority</td>
+                            <td className="text-lg-end">
+                                <Address pubkey={authority} alignRight raw />
+                            </td>
+                        </tr>
+                        <tr>
+                            <td>Creation Slot</td>
+                            <td className="text-lg-end">
+                                <span className="text-monospace">{creationSlot.toString()}</span>
+                            </td>
+                        </tr>
+                        <tr>
+                            <td>Max Depth</td>
+                            <td className="text-lg-end">
+                                <span className="text-monospace">{treeHeight}</span>
+                            </td>
+                        </tr>
+                        <tr>
+                            <td>Max Buffer Size</td>
+                            <td className="text-lg-end">
+                                <span className="text-monospace">{maxBufferSize}</span>
+                            </td>
+                        </tr>
+                        <tr>
+                            <td>Canopy Depth</td>
+                            <td className="text-lg-end">
+                                <span className="text-monospace">{canopyDepth}</span>
+                            </td>
+                        </tr>
+                        <tr>
+                            <td>Current Sequence Number</td>
+                            <td className="text-lg-end">
+                                <span className="text-monospace">{seq.toString()}</span>
+                            </td>
+                        </tr>
+                        <tr>
+                            <td>Current Root</td>
+                            <td className="text-lg-end">
+                                <Address pubkey={new PublicKey(root)} alignRight raw />
+                            </td>
+                        </tr>
+                        <tr>
+                            <td>Next Leaf Index</td>
+                            <td className="text-lg-end">
+                                <span className="text-monospace">{rightMostIndex}</span>
+                            </td>
+                        </tr>
+                        <tr>
+                            <td>Remaining Leaves</td>
+                            <td className="text-lg-end">
+                                <span className="text-monospace">{Math.pow(2, treeHeight) - rightMostIndex + 1}</span>
```

Comment:
maybe one more number which is the max capacity of the tree so i dont have to do more math

Reply:
How about `Total Possible Leaves` ?